tools/install.py

```diff
@@ -142,8 +157,29 @@ def files(action):
     else:
       output_file = 'lib' + output_file + '.' + variables.get('shlib_suffix')
 
+    # GYP will output to lib.target except on OS X, this is hardcoded
```

Comment:
I don't understand the OS X comment here for a section that is zos specific?

Reply:
I think this comment was left in by mistake. I have removed it.